scripts/launch-s0-instance.sh

```diff
@@ -3,81 +3,233 @@
-    echo "   $me --k /path/to/key.pem -n my_s0_server -t m5d.2xlarge"
+    echo "   $me --ssh-key-name my_ec2_key --instance-name my_s0_server --instance-type m5d.2xlarge --kms-key-alias my_kms_key --bucket-name my_s3_bucket"
+    echo "   $me -s my_ec2_key -n my_s0_server -t m5ad.2xlarge -k my_kms_key -b my_s3_bucket"
+    echo
+    echo "  -s | --ssh-key-name       : A name of an aws EC2 key in the PEM format for the SSH connection to the instance."
+    echo
+    echo "  -n | --instance-name      : Optional. Name of an instance. It's shown in the AWS console."
+    echo "                              The default value is \"[user_name]-s0-server\"."
+    echo
+    echo "  -t | --instance-type      : Optional. An instance type. See https://aws.amazon.com/ec2/instance-types/ for more details."
+    echo "                              The default value is \"m5d.2xlarge\"."
+    echo
+    echo "  -g | --security-group     : Optional. A name of the security group which has ports 22 and 61234 open to the world for SSH and s0 server respectively."
+    echo "                              The grop 'elastio-s0-server' will be created and then used in case if this OR next argument is not specified."
+    echo "                              NOTE: The group existance and the pen ports in the group aren't checked."
+    echo "                                    The script may not have permissions and assumes that the group is configured properly."
     echo
-    echo "  -k | --key-name : A name of an aws EC2 key in the PEM format for the SSH connection to the instance."
+    echo "  -b | --bucket-name        : s3 bucket name for s0 vault."
     echo
-    echo "  -n | --name     : Optional. Name of an instance. It's shown in the AWS console."
-    echo "                    The default value is \"[user_name]-s0-server\"."
+    echo "  -k | --kms-key-alias      : KMS key alias for the data encription in the s0 vault."
     echo
-    echo "  -t | --type     : Optional. An instance type. See https://aws.amazon.com/ec2/instance-types/ for more details."
-    echo "                    The default value is \"m5d.2xlarge\"."
+    echo "  -p | --instance-profile   : An instance provile with access to the s3 bucket and KMS key from 2 parametrers abowe."
+    echo "                              See aws doscs for more details how to create one:"
+    echo "                              https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html"
+    echo "                              https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html"
     echo
-    echo "  -h | --help     : Show this usage help."
+    echo "  -z | --s0-block-size      : Optional. A block size for the s0 shard. There are 2 possible values: \"fixed\" and \"variable\". The \"fixed\" is default."
+    echo "                              - The \"fixed\" selects the shard with a fixed-length block size."
```

Comment:
```suggestion
    echo "                              - The \"fixed\" selects the shard with using fixed-block deduplication."
```

Reply:
Maybe `with` is excess? Just `The \"fixed\" selects the shard using fixed-block deduplication.` ?